documentation/asciidoc/stories/assembly_getting_started.adoc

```diff
@@ -2,7 +2,10 @@
 :context: starter_intro
 
 include::{topics}/con_sb_starter.adoc[leveloffset=+1]
+//Community Only
+ifndef::productized[]
 include::{topics}/proc_clone_repo.adoc[leveloffset=+1]
```

Comment:
can be removed? I don't see why the users need to clone this repository :/ (cc @karesti )

Reply:
let's remove it.